index.html

```diff
@@ -201,5 +201,88 @@ <h3 class="white">Opieka nad grobami</h3>
         </p>
       </div>
     </section>
+    <section id="speaker">
+      <div class="section-row">
+        <div class="section-white">
+          <p class="text-black">
+            Współpracujemy z doświadczonym Mówcą Pogrzebowym, specjalizującym się w prowadzeniu <strong>modlitw w kaplicy pogrzebowej</strong> 
+oraz organizacji <strong>świeckich ceremonii pogrzebowych</strong>. 
+          </p>
+        </div>
+        <div class="section-white">
+          <div class="section-right-align">
+            <button>
+              <p class="button-text">MÓWCA POGRZEBOWY</p>
+            </button>
+          </div>
+        </div>
+      </div>
+    </section>
+
+
+
+    <section id="contact">
+      <div id="contact-container">
+ 
+          <h3 class="contact-header">Kontakt</h3>
+
+
+       <div id="contact-two-columns"> 
+
+
+            <div id="contact-left">
```

Comment:
I can see many `id` attributes - do we need all of them? As a rule of thumb, it's always better not to overuse ids. For styling, classes and tags should be enough in most of cases. :)

Reply:
I removed most of ID's :)

Reply:
And I used BEM a little.

Reply:
Perfect <3